tests/serializers/test_json.py

```diff
@@ -319,3 +335,23 @@ def test_timedelta(self):
             json.dumps({'duration': duration}, cls=DjangoJSONEncoder),
             '{"duration": "P0DT00H00M00S"}'
         )
+
+
+class DumpdataUnicodeTestCase(TestCase):
+    def test_allow_unicode(self):
+        Actor.objects.create(name='יוניקוד')
+
+        tests = (  # (allow_unicode, expected)
+            (False, '\\u05d9\\u05d5\\u05e0\\u05d9\\u05e7\\u05d5\\u05d3'),
+            (True, 'יוניקוד')
+        )
+        for allow_unicode, expected in tests:
+            with self.subTest(allow_unicode=allow_unicode, expected=expected):
+                out = StringIO()
+                call_command(
+                    'dumpdata',
+                    'serializers.actor',
+                    allow_unicode=allow_unicode,
+                    stdout=out,
+                )
+                self.assertTrue(expected in out.getvalue().strip())
```

Comment:
Can use `asserIn` here for a more specific assertion method. Same for other tests.

```suggestion
                self.assertIn(expected, out.getvalue().strip())
```